pom.xml

```diff
@@ -141,6 +150,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- TODO remove spring version, it will come from the core -->
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+      <version>${springVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jcl</artifactId>
+      <version>${springVersion}</version>
+    </dependency>
```

Comment:
Per offline conversation with @jtnord let's try just excluding and using the one coming from jenkins-core

Reply:
Done !!